internal/clients/team4/client.go

```diff
@@ -16,51 +18,102 @@ func init() {
 	baseclient.RegisterClientFactory(id, func() baseclient.Client { return NewClient(id) })
 }
 
-// NewClient is a function that creates a new empty client
-func NewClient(clientID shared.ClientID) baseclient.Client {
-	iigoObs := &iigoObservation{
+func newClientInternal(clientID shared.ClientID, testing *testing.T) client {
```

Comment:
that's a very strange way of testing :O, is it not possible to unit test the judge separately? Including the `testing` package in production is something that rarely happens.

Reply:
ignore for now. would love to see a PR resolving this. tracking in #386